Tatsi/Views/Assets Grid/Cells/AssetCollectionViewCell.swift

```diff
@@ -117,6 +120,10 @@ final internal class AssetCollectionViewCell: UICollectionViewCell {
             return
         }
         self.shouldUpdateImage = false
+        
+        // Set the correct checkmark color
+        self.contentView.subviews[2].subviews[0].tintColor = self.colors?.checkMark ?? TatsiConfig.default.colors.checkMark
```

Comment:
Using an index on subviews to find a view is bit risky. Users could add a view to the contentView and thus break this code or even crash the app. My suggestion would be to define the iconView as a private property on `AssetCollectionViewCell` and reference to that when setting the color. 

(You will have to manually apply this suggestion higher in the file)
```suggestion
        lazy private var selectedOverlay: UIView = {
        let view = UIView()
        view.translatesAutoresizingMaskIntoConstraints = false
        view.isHidden = true
        view.backgroundColor = UIColor.white.withAlphaComponent(0.4)
        
        view.addSubview(self.iconView)
        view.bottomAnchor.constraint(equalTo: self.iconView.bottomAnchor, constant: 3).isActive = true
        view.trailingAnchor.constraint(equalTo: self.iconView.trailingAnchor, constant: 3).isActive = true
        
        return view
    }()
    
    lazy private var iconView: SelectionIconView = {
        let iconView = SelectionIconView()
        iconView.translatesAutoresizingMaskIntoConstraints = false
        return iconView
    }()
```

Also, this is a little known thing but `.tintColor` on UIView allows a nil value (yes even in Swift where it is force-unwrapped). Supply a nil value for tint color will make the UIView search for a tint color in the view hierarchy (so it's asks the parent for a color). This allows the view to participate in the tint color system. 

To keep consitency with older versions of this library I would recommend this for setting the color:
```suggestion
        self.iconView.tintColor = self.colors?.checkMark
```

Reply:
Implemented the suggestion